Adds overlay list paths specific to host vmware.sciencesoft.at.

Index: layman/layman.cfg
===================================================================
--- layman.orig/layman.cfg
+++ layman/layman.cfg
@@ -32,6 +32,9 @@ make_conf : %(storage)s/make.conf
 #            file:///usr/portage/local/layman/my-list.xml
 
 overlays  : http://www.gentoo.org/proj/en/overlays/layman-global.txt
+            http://rpo.xw.ath.cx/gentoo/portage/overlays/foreign.xml
+            http://rpo.xw.ath.cx/gentoo/portage/overlays/hosted.xml
+            http://rpo.xw.ath.cx/gentoo/portage/overlays/xworld.xml
 
 #-----------------------------------------------------------
 # Proxy support
